### PR TITLE
Implement get functionality for Follows Servlet

### PR DESCRIPTION
**Pull request by lukasberglund**
This is so that you can query who a user follows or what users follow a given business. There can only be exactly one argument in a query and it must be either 'userId' or 'businessId'.

### CODE REVIEW

File: capstone/src/main/java/com/google/sps/servlets/follow/FollowServlet.java [hunk 1]
```diff
@@ -156,4 +157,25 @@ public void doDelete(HttpServletRequest request, HttpServletResponse response)
 
     response.sendRedirect("/business.html?id=" + businessId);
   }
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    User currentUser = userService.getCurrentUser();
+    if (currentUser == null) {
+      response.sendError(
+          HttpServletResponse.SC_UNAUTHORIZED, "User must be logged in to unfollow a business.");
```
Comment (prabenzom) [inline capstone/src/main/java/com/google/sps/servlets/follow/FollowServlet.java:166]: unfollow?

Comment (lukasberglund) [reply capstone/src/main/java/com/google/sps/servlets/follow/FollowServlet.java:166]: Ah, I missed that when copy pasting. Thanks for noticing!!

File: capstone/src/main/java/com/google/sps/servlets/follow/FollowServlet.java [hunk 2]
```diff
@@ -163,7 +163,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     User currentUser = userService.getCurrentUser();
     if (currentUser == null) {
       response.sendError(
-          HttpServletResponse.SC_UNAUTHORIZED, "User must be logged in to unfollow a business.");
+          HttpServletResponse.SC_UNAUTHORIZED, "User must be logged in to make a get request.");
```
Comment (eash130) [inline capstone/src/main/java/com/google/sps/servlets/follow/FollowServlet.java:166]: "user must be logged in to view followed businesses

File: capstone/src/main/java/com/google/sps/servlets/follow/FollowsServlet.java [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    List<String> requestFilters =
+        Arrays.stream(POSSIBLE_FILTER_PARAMETERS)
+            .filter(parameter -> request.getParameter(parameter) != null)
+            .collect(Collectors.toList());
+
+    if (requestFilters.size() != 1) {
+      response.sendError(
+          HttpServletResponse.SC_BAD_REQUEST,
+          "Must specify either businessId or userId, but not both.");
```
Comment (winniewan) [inline capstone/src/main/java/com/google/sps/servlets/follow/FollowsServlet.java:60]: There could be "none" too, right?

Comment (lukasberglund) [reply capstone/src/main/java/com/google/sps/servlets/follow/FollowsServlet.java:60]: I am not quite sure what you mean. Could  you elaborate?

Comment (prabenzom) [reply capstone/src/main/java/com/google/sps/servlets/follow/FollowsServlet.java:60]: The user could have specified no id?

**Pull request merged**